Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -1357,7 +1357,7 @@ static Class _WebMessageDocumentClass()
 void HTMLConverter::_addQuoteForElement(Element& element, BOOL opening, NSInteger level)
 {
     unichar c = ((level % 2) == 0) ? (opening ? 0x201c : 0x201d) : (opening ? 0x2018 : 0x2019);
-    RetainPtr<NSString> string = adoptNS([[NSString alloc] initWithCharacters:&c length:1]);
+    RetainPtr<NSString> string = adoptNS([[NSString alloc] initWithCharactersNoCopy:&c length:1 freeWhenDone:NO]);
```

Comment:
Same here; the string is created with a pointer to a stack variable. I would like to see this addressed before landing.